src/cryptography/hazmat/primitives/asymmetric/utils.py

```diff
@@ -28,6 +29,9 @@ def decode_rfc6979_signature(signature):
         raise ValueError(
             "The signature contains bytes after the end of the ASN.1 sequence."
         )
+    if data == eoo.endOfOctets:
```

Comment:
Done. It's harmless, so unless you'd prefer to include text about removing it, it seems more robust to keep it indefinitely. If you want to add something about removing it, I can add a comment after I file a bug (so we can link to the bug).